tests/testthat/_snaps/render_html.md

```diff
@@ -394,4 +394,3 @@
       <p>This should be</p>
       </div>
       </div>
-
```

Comment:
The tests are reporting that there is now an extra `</div>` element here. I'll take a look at that.

Reply:
Ultimately it was fine. This was an artifact of my masking some of the output due to the fact that pandoc does not output attributes in a specific order.